runtime-modules/utility/Cargo.toml

```
@@ -1,32 +1,33 @@
 [package]
 name = 'pallet-utility'
-version = '1.1.0'
+version = '5.0.0'
```

Incorrect version update
```suggestion
version = '2.0.0'
```